pre_commit/main.py

```diff
@@ -167,6 +167,12 @@ def main(argv=None):
             'tagged version (the default behavior).'
         ),
     )
+    autoupdate_parser.add_argument(
+        '--repo', nargs=1, default=None,
+        help=(
+            'Repository to update the hooks of.'
+        ),
+    )
```

Comment:
the default for `nargs` is `1` and the default for `default` is None (so no need to specify them

but I guess that brings up the question: should I be able to do:

```
pre-commit autoupdate --repo repo1 --repo repo2 --repo repo3
```

to update multiple repositories?

That can be implemented pretty easily with `action='append'` -- thoughts?

Reply:
Since it doesn't match my use-case I'm not very motivated to, to be honest. We can make an issue for future improvement though, `good-first-issue` or something.

Reply:
good enough for me!